package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "2024-aug-fst-recipe-finder-be",
+  "version": "1.0.0",
+  "description": "project template repo",
+  "main": "main.ts",
+  "scripts": {
+    "test": "jest",
+    "dev": "nodemon --ext ts --exec ts-node main.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/iO-Academy/2024-aug-fst-recipe-finder-BE.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/iO-Academy/2024-aug-fst-recipe-finder-BE/issues"
+  },
+  "homepage": "https://github.com/iO-Academy/2024-aug-fst-recipe-finder-BE#readme",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.21.2",
+    "node": "^22.12.0",
+    "promise-mysql": "^5.2.0"
+  },
+  "devDependencies": {
+    "@types/cors": "^2.8.17",
+    "@types/express": "^5.0.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^22.10.1",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.7.2"
+  }
```

Comment:
Thinking about it, since your npm run script depends on nodemon and ts-node, it would probably be best to have these added to the project as dev dependencies

`npm install --save-dev nodemon ts-node`

Reply:
tick